pkg/agent/openflow/pipeline.go

```diff
@@ -41,39 +41,75 @@ import (
-	ConntrackCommitTable         = binding.NewOFTable(105, "ConntrackCommit")
-	ServiceConntrackCommitTable  = binding.NewOFTable(106, "ServiceConntrackCommit")
-	HairpinSNATTable             = binding.NewOFTable(108, "HairpinSNAT")
-	L2ForwardingOutTable         = binding.NewOFTable(110, "Output")
+	// PipelineClassifierTable is table 0. Packets are forwarded to different pipelines in this table.
+	PipelineClassifierTable = newFeatureTable("PipelineClassifier", 0xff, 0)
+
+	// Pipeline of ARP is used to process ARP packets.
+	// ValidationStage
+	ARPSpoofGuardTable = newFeatureTable("ARPSpoofGuard", 0x7f, binding.ValidationStage)
+
+	// OutputStage
+	ARPResponderTable = newFeatureTable("ARPResponder", 0x7f, binding.OutputStage)
+
+	// Pipeline of IP is used to process IPv4/IPv6 packets.
+	// ClassifierStage
+	ClassifierTable = newFeatureTable("Classification", 0x7f, binding.ClassifierStage)
+	UplinkTable     = newFeatureTable("Uplink", 0x6f, binding.ClassifierStage)
+
+	// ValidationStage
+	SpoofGuardTable          = newFeatureTable("SpoofGuard", 0x7f, binding.ValidationStage)
+	IPv6Table                = newFeatureTable("IPv6", 0x6f, binding.ValidationStage)
+	MulticastClassifierTable = newFeatureTable("MulticastClassifier", 0x5f, binding.ValidationStage)
+
+	// ConntrackStateStage
+	SNATConntrackTable  = newFeatureTable("SNATConntrackZone", 0x7f, binding.ConntrackStateStage)
+	ConntrackTable      = newFeatureTable("ConntrackZone", 0x6f, binding.ConntrackStateStage)
+	ConntrackStateTable = newFeatureTable("ConntrackState", 0x5f, binding.ConntrackStateStage)
+
+	// EgressSecurityStage
+	AntreaPolicyEgressRuleTable = newFeatureTable("AntreaPolicyEgressRule", 0x7f, binding.EgressSecurityStage)
+	EgressRuleTable             = newFeatureTable("EgressRule", 0x6f, binding.EgressSecurityStage)
+	EgressDefaultTable          = newFeatureTable("EgressDefaultRule", 0x5f, binding.EgressSecurityStage)
+	EgressMetricTable           = newFeatureTable("EgressMetric", 0x4f, binding.EgressSecurityStage)
+
+	// PreRoutingStage
+	// AntreaProxy is enabled.
+	PreRoutingClassifierTable = newFeatureTable("PreRoutingClassifier", 0x8f, binding.PreRoutingStage)
+	NodePortProbeTable        = newFeatureTable("NodePortProbe", 0x7f, binding.PreRoutingStage)
+	SessionAffinityTable      = newFeatureTable("SessionAffinity", 0x6f, binding.PreRoutingStage)
+	ServiceLBTable            = newFeatureTable("ServiceLB", 0x5f, binding.PreRoutingStage)
+	EndpointDNATTable         = newFeatureTable("EndpointDNAT", 0x4f, binding.PreRoutingStage)
+	// AntreaProxy is disabled.
+	DNATTable = newFeatureTable("DNAT", 0x7f, binding.PreRoutingStage)
+
+	// RoutingStage
+	L3ForwardingTable       = newFeatureTable("L3Forwarding", 0x7f, binding.RoutingStage)
+	ServiceHairpinMarkTable = newFeatureTable("ServiceHairpinMark", 0x6f, binding.RoutingStage)
+	L3DecTTLTable           = newFeatureTable("IPTTLDec", 0x5f, binding.RoutingStage)
```

Comment:
better to make the name format consistent as others, if it's `L3DecTTLTable`, then the name should be `L3DecTTL`

Reply:
Agreed, thanks!